scripts/generate_oks_session

```diff
@@ -84,15 +89,70 @@ def generate(bootfile, datafile, session_name):
     vhostdals[h] = vhdal
 
   # ###########################################################################
-  # Create a run controller for the segment. We say it runs on the last
-  # specified virtual host for now since this is not defined in the json
+  # Create a run controller for the segment. 
   # ###########################################################################
+  fsm = controller['statefulnode']['fsm']
+  for iface in fsm['interfaces']:
+    ifname = 'user-providedirun-number'
```

Comment:
user-providedirun-number -> user-provided-run-number

Reply:
Actually, you don't need this one right? its filled automatically

Reply:
Yeah, I think that is a mistake leftover from when I was just inventing a configuration on the fly rather than converting from your controller.json. I will fix this